### PR TITLE
dvclive: use step-based images in how-it-works

### PR DESCRIPTION
**Pull request by dberenbaum**
Follow up per https://github.com/iterative/dvc.org/pull/4750#pullrequestreview-1573979908

### CODE REVIEW

File: content/docs/dvclive/how-it-works.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ To illustrate with an example, given the following code:
 
 ```python
 import random
+from pathlib import Path
```
Comment (daavoo) [inline content/docs/dvclive/how-it-works.md:31]: No needed?
```suggestion
```

**Review by daavoo — APPROVED** [review]

File: content/docs/dvclive/how-it-works.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/dvclive/how-it-works.md:31]: I don't follow. The code snippet doesn't work without it, and other imports are included. Why include `import random` but not this?

File: content/docs/dvclive/how-it-works.md [hunk 2]
```diff
@@ -40,7 +41,7 @@ with Live(save_dvc_exp=True) as live:
     for i in range(EPOCHS):
         live.log_metric("metric", i + random.random())
         live.log_metric("nested/metric", i + random.random())
-        live.log_image("img.png", Image.new("RGB", (50, 50), (i, i, i)))
+        live.log_image(f"img/{live.step}.png", Image.new("RGB", (50, 50), (i, i, i)))
         Path("model.pt").write_text(str(random.random()))
```
Comment (dberenbaum) [inline content/docs/dvclive/how-it-works.md:45]: @daavoo Here's where the import statement comes from (sorry that it's unrelated)

Comment (daavoo) [reply content/docs/dvclive/how-it-works.md:45]: I see

**Pull request merged**